sdcflows/workflows/base.py

```diff
@@ -142,21 +144,19 @@ def init_sdc_wf(boldref, omp_nthreads=1, debug=False, ignore=None):
         ignore = tuple(ignore)
 
     fmaps = defaultdict(list, [])
-    for associated in boldref.get_associations(kind='InformedBy'):
-        if associated.suffix == 'epi':
+    for associated in distorted_ref.get_associations(kind='InformedBy'):
+        if associated['suffix'] in ('epi', 'phasediff', 'fieldmap', 'phase1', 'phase2'):
```

Comment:
why the change to indexed access? Isn't pybids deprecating it?
```suggestion
        if associated.suffix in ('epi', 'phasediff', 'fieldmap', 'phase1', 'phase2'):
```